Automattic/jetpack
# Protect: Add/empty status

#### Changes proposed in this Pull Request:

* Creates the empty status, when the plugin does not have any information yet.

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Connect and go to Protect
* Change your response_status to empty in your wp-config:
```
define( 'JETPACK_PROTECT_DEV__BYPASS_CACHE', true );
define( 'JETPACK_PROTECT_DEV__API_RESPONSE_TYPE', 'empty' );
```
* Refresh the page and see the new screen

Note 1: The original layout has a progress bar we will not implement - this will still need design review so the top margin in the Header is a temporary fix
Note 2: There are some issues in small viewports that will be handled in another PR. Same thing for the Cols with grid value zero. This is also being mistakenly used in the Footer and will be fixed later

![Captura de tela de 2022-04-28 19-07-17](https://user-images.githubusercontent.com/971483/165855689-4d3d9ede-ca98-4ea0-bbc9-4c5bc1bc8517.png)



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202183900595806
